src/EncompassRest/ClientParameters.cs

```diff
@@ -70,6 +70,11 @@ public int TimeoutRetryCount
         /// </summary>
         public UndefinedCustomFieldHandling UndefinedCustomFieldHandling { get; set; }
 
+        /// <summary>
+        /// The URL to call for API calls.Defaults to "https://api.elliemae.com/".
+        /// </summary>
+        public string BaseAddress { get; set; } = "https://api.elliemae.com/";
```

Comment:
By default could we leave it as `null` and then check for `null` in the `EncompassRestClient` constructor and null coalesce the default value of `https://api.elliemae.com/`? I prefer that since `ClientParameters` is json serializable and don't want to serialize the default value.

Reply:
Good catch, I forgot about that.  Changes were made.